xml/System/MemoryExtensions.xml

```diff
@@ -20,7 +20,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Extension methods for Span{T}, Memory{T}, and friends.</summary>
```

Comment:
I'm fine with the language personally, but I'm not sure how formal docs need to be and whether "and friends" qualifies.